internal/service/pushbasedlogexport/resource_migration_test.go

```diff
@@ -0,0 +1,68 @@
+package pushbasedlogexport_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
+)
+
+func TestMigPushBasedLogExport_basic(t *testing.T) {
+	mig.SkipIfVersionBelow(t, "1.16.0")
```

Comment:
Recommend to use mig.CreateAndRunTest so acc and mig tests are defined in one place, e.g.:
https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/service/backupcompliancepolicy/resource_backup_compliance_policy_migration_test.go#L10